tests/test_endpoints.py

```diff
@@ -74,18 +74,14 @@ def setUp(self):
         self.relations = {
             'test-endpoint': [
                 {
-                    'local': {'app-key': 'value'},
-                    'local/0': {'key': 'value'},
-                    'unit': {'simple': 'value', 'complex': '[1, 2]'},
```

Comment:
This is where the complex component of the app data bag is removed - this seems somewhat adjunct to the renaming of the units to match the remote application name?